docs/static/gamejam/lib/gamejam.js

```diff
@@ -21,15 +21,15 @@ function initTelemetry() {
 function initRulesTelemetry(id) {
     var _a, _b, _c;
     // Try one of our tutorials text link
-    (_a = document.querySelector("#" + id + " + ul li:nth-child(1) a")) === null || _a === void 0 ? void 0 : _a.addEventListener("click", function () {
+    (_a = document.querySelector("#".concat(id, " + ul li:nth-child(1) a"))) === null || _a === void 0 ? void 0 : _a.addEventListener("click", function () {
         window.pxtTickEvent("gamejam.link.tutorial");
     });
     // How to import images text link
-    (_b = document.querySelector("#" + id + " + ul li:nth-child(3) a")) === null || _b === void 0 ? void 0 : _b.addEventListener("click", function () {
+    (_b = document.querySelector("#".concat(id, " + ul li:nth-child(3) a"))) === null || _b === void 0 ? void 0 : _b.addEventListener("click", function () {
         window.pxtTickEvent("gamejam.link.images");
     });
     // Developer documentation text link
-    (_c = document.querySelector("#" + id + "+ ul li:nth-child(4) a")) === null || _c === void 0 ? void 0 : _c.addEventListener("click", function () {
+    (_c = document.querySelector("#".concat(id, "+ ul li:nth-child(4) a"))) === null || _c === void 0 ? void 0 : _c.addEventListener("click", function () {
```

Comment:
Just curious about the reason for changing all of the "+" to `.concat()`? Is the function better so you avoid possible arithmetic?

Reply:
this is in the compiled javascript, there's no tsconfig in this folder so the default settings for the typescript compiler probably changed.